Tests/SonarScanner.MSBuild.Common.UnitTests/MsBuildPathSettingsTests.cs

```diff
@@ -51,20 +50,21 @@ public void GetImportBeforePaths_NonWindows_User_Account()
                 (Environment.SpecialFolder.UserProfile, "c:\\user profile"),
             };
 
-            var settings = new MsBuildPathSettings(GetFolderPath(paths), IsWindows(false), FileExists(true));
+            var settings = new MsBuildPathSettings(GetFolderPath(paths), IsWindows(false), DirExists(true));
 
             var result = settings.GetImportBeforePaths();
 
             result.Should().HaveCount(7);
             result.Should().Contain(
-                Path.Combine("c:\\app data", "Microsoft", "MSBuild", "4.0", "Microsoft.Common.targets", "ImportBefore"),
-                Path.Combine("c:\\app data", "Microsoft", "MSBuild", "10.0", "Microsoft.Common.targets", "ImportBefore"),
-                Path.Combine("c:\\app data", "Microsoft", "MSBuild", "11.0", "Microsoft.Common.targets", "ImportBefore"),
-                Path.Combine("c:\\app data", "Microsoft", "MSBuild", "12.0", "Microsoft.Common.targets", "ImportBefore"),
-                Path.Combine("c:\\app data", "Microsoft", "MSBuild", "14.0", "Microsoft.Common.targets", "ImportBefore"),
-                Path.Combine("c:\\app data", "Microsoft", "MSBuild", "15.0", "Microsoft.Common.targets", "ImportBefore"),
-                Path.Combine("c:\\use profile", "MSBuild", "15.0", "Microsoft.Common.targets", "ImportBefore")
-                );
+                new[] {
```

Comment:
The way the params were being passed meant only the first item was being checked.
The other items were being treated as message arguments.